lib/rules/no-this-translate.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @fileoverview Disallow the use of this.translate
+ * @author Automattic
+ * @copyright 2016 Automattic. All rights reserved.
+ * See LICENSE.md file in root directory for full license.
+ */
+'use strict';
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+const ERROR_MESSAGE = 'Use localize( ReactComponent ) instead of this.translate';
+
+module.exports = {
+	meta: {
+		docs: {
+			description: 'Disallow the use of this.translate',
+			category: 'Deprecation',
+			recommended: true
+		},
+		fixable: 'code',
```

Comment:
Per rule docs, we should omit `fixable` because it's not a fixable issue:

>Omit the `fixable` property if the rule is not fixable.

http://eslint.org/docs/developer-guide/working-with-rules#rule-basics